### PR TITLE
Add fullscreen with support to remote presentation controller

### PR DESCRIPTION
**Pull request by giflw**
- F5 to enter/exit
- Escape to exit

### CODE REVIEW

File: src/plugins/fullscreen/fullscreen.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    // Wait for impress.js to be initialized
+    document.addEventListener( "impress:init", function( event ) {
+        var api = event.detail.api;
+        var root = event.target;
+        var gc = api.lib.gc;
+        var util = api.lib.util;
+
+        gc.addEventListener( document, "keydown", function( event ) {
```
[the remark this answers is not in the document]
Comment (giflw) [reply src/plugins/fullscreen/fullscreen.js]: Done changing to only one event listener. F5 indeed enter and exit full screen, as you requested earlier. It makes the behavior more intuitive when using keyboard, but allows the use of remote controller

[the remark this answers is not in the document]
Comment (aspiers) [reply src/plugins/fullscreen/fullscreen.js]: FWIW, F5 is used as the reload key in some popular browsers.

[the remark this answers is not in the document]
Comment (henrikingo) [reply src/plugins/fullscreen/fullscreen.js]: That's a good point, and this code will now override that functionality.

I think compatibility with remote controllers is a good argument in favor though. Other than that I have no strong opinion.

[the remark this answers is not in the document]
Comment (giflw) [reply src/plugins/fullscreen/fullscreen.js]: I think it's more intuitive to use F5 in presentations to enter fullscreen. I catch myself trying that some times. Besides that, remote controllers works as keyboards and send F5 key to enter fullscreen. This way impress.js becomes more friendly to power point and other common users that have remote controls.
Regarding browser reload, CTRL+R it's another common way to reload, an this is not changed by impress.js.

**Pull request merged**